terraform/modules/configs/deploy-all/variables.tf

```diff
@@ -31,6 +31,16 @@ variable "ecr_image_id_client" {
   default = "latest"
 }
 
+variable "ecr_image_id_s3_virus_scan" {
+  type    = string
+  default = "latest"
+}
+
+variable "ecr_image_id_s3_virus_scan" {
```

Comment:
Duplicate variable here - throws an error on `terraform plan`

Reply:
this should be fixed now